src/zeroband/utils/config.py

```diff
@@ -6,6 +6,32 @@
 from zeroband.utils.pydantic_config import BaseConfig
 
 
+class ModelConfig(BaseConfig):
+    """Configures the model to be used for training."""
+
+    name: Annotated[
+        str,
+        Field(
+            default="Qwen/Qwen3-0.6B",
+            description="Name or path of the HF model to use.",
+        ),
+    ]
+
+
+class PathConfig(BaseConfig):
+    """Configures a path used for input/ output operations"""
+
+    path: Annotated[Path, Field(description="Path to write to.")]
+
+    clean: Annotated[
+        bool,
+        Field(
+            default=False,
+            description="Whether to clean the path at the beginning of the run. If True, will delete the entire directory.",
+        ),
+    ]
+
```

Comment:
I don't liek this PathConfig, this is too much, lets just duplicate the path and clean on most path